cli/src/extension_api.ts

```diff
@@ -1,395 +1,402 @@
+   * restrict this execution environment when dealing with external commands
+   * that could potentially misbehave. This API allows restricting
+   * filesystem and network access for those processes.
+   *
+   * @param process - The command which shall be executed and its restrictions
+   *
+   * Process example:
+   * ```
+   * {
+   *   cmd: "ls",
+   *   args: ["-lah"],
+   *   stdin: "null",
+   *   stdout: "piped",
+   *   stderr: "inherit",
+   *   exceptions: {
+   *     read: ["~/"],
+   *     write: false,
+   *     run: ["ls"],
+   *     net: false,
+   *     strict: false,
+   *   }
+   * }
+   * ```
+   *
+   * The `read`/`write`/`run` permissions accept either an array of paths,
+   * or a boolean. Paths must either be absolute or start with `~/`.
+   *
+   * For `run` the executables will be resolved from `$PATH` when they are
+   * neither absolute nor start with `~/`.
+   *
+   * The `net` permission accepts only boolean values.
+   *
+   * Some exceptions are added by default, to simplify the extension creation
+   * process. If you're looking for more granular control, you can set strict
+   * to `true` and no exceptions will be added without explicitly specifying
+   * them.
+   *
+   * @return Process status and output
+   *
+   * Process status and output example:
+   * ```
+   * {
+   *   stdout: "Hello, World!",
+   *   stderr: "",
+   *   success: true,
+   *   code: 0,
+   * }
+   * ```
+   *
+   * If the process got killed by a signal, it will contain a `signal` field
+   * instead of `code`:
+   *
+   * ```
+   * {
+   *   stdout: "",
+   *   stderr: "Getting killed by signal...",
+   *   success: false,
+   *   signal: 31,
+   * }
+   * ```
+   */
   static runSandboxed(process: object): object {
-    return Deno.core.opSync("run_sandboxed", process);
+    return opSync("run_sandboxed", process);
   }
 
-  /// Get the extension's manifest permissions.
-  ///
-  /// # Returns
-  ///
-  /// The permissions object:
-  /// ```
-  /// {
-  ///   read: ["~/.npm"],
-  ///   write: ["/tmp"],
-  ///   run: ["ls", "echo", "npm"],
-  ///   env: ["HOME", "PATH"],
-  ///   net: false
-  /// }
-  /// ```
+  /**
+   * Get the extension's manifest permissions.
+   *
+   * @returns Extension permissions
+   *
+   * Permissions object example:
+   * ```
+   * {
+   *   read: ["~/.npm"],
+   *   write: ["/tmp"],
+   *   run: ["ls", "echo", "npm"],
+   *   env: ["HOME", "PATH"],
+   *   net: false
+   * }
+   * ```
+   */
   static permissions(): object {
-    return Deno.core.opSync("op_permissions");
+    return opSync("op_permissions");
   }
 }
+
+function opSync<T>(op: string, ...args: unknown[]): T {
+  // deno-lint-ignore no-explicit-any
+  return (Deno as any).core.opSync(op, ...args) as T;
+}
+
+function opAsync<T>(op: string, ...args: unknown[]): Promise<T> {
+  // deno-lint-ignore no-explicit-any
+  return (Deno as any).core.opAsync(op, ...args) as Promise<T>;
+}
```

Comment:
I'd assume it was complaining that `Deno.core` didn't exist and this was the easiest solution to solve it?

Reply:
Yep. That's exactly what happened.